kebechet/managers/version/release_triggers.py

```diff
@@ -40,13 +40,42 @@
 class BaseTrigger:
     """Base class for triggers of new version releases."""
 
+    def adjust_version_toml_file(self, file_path: str) -> Optional[tuple]:
+        """Adjust version in the toml file, return signalizes whether the return value indicates change in file."""
+        changed = False
+        with open(file_path, "r") as f:
+            toml_content = toml.loads(f.read())
+        project = toml_content.get("project", None)
+        old_version = project.get("version", None)
+        try:
+            new_version = self.get_new_version(old_version)
+        except VersionError as e:
+            raise VersionError(
+                UNABLE_TO_UPDATE_VERSION_ERROR.format(
+                    file_path=file_path,
+                    line_num=1,
+                    old_version=old_version,
+                    reason=str(e),
+                    environment_details=ManagerBase.get_environment_details(),
+                )
+            ) from e
+        _LOGGER.info("Computed new version: %s", new_version)
+        if old_version:
```

Comment:
This check should occur above and just return if `version` exists in `pyproject.toml`

Reply:
moved it above, thanks